src/Driver/FlatFile/Driver.php

```diff
@@ -158,7 +161,9 @@ public function peekQueue($queueName, $index = 0, $limit = 20)
         $files = array_keys(iterator_to_array($it));
 
         natsort($files);
-        $files = array_reverse($files);
+        if ($this->queueType === 'life') {
```

Comment:
typo ?

Reply:
yep, not sure why all tests pass with it :(

Reply:
Because there is no specific test for the new fifo/lifo functionality?
(i created a pull request with fix and tests against your branch)